packages/platform-ios/native_modules.rb

```diff
@@ -5,12 +5,20 @@
 #
 def use_native_modules!(root = "..", packages = nil)
   if (!packages)
-    # Resolve the CLI's main index file
-    cli_bin = Pod::Executable.execute_command("node", ["-e", "console.log(require.resolve('@react-native-community/cli/build/index.js'))"], true).strip
+    command = "node"
+    args = ["./node_modules/.bin/react-native", "config"]
+    begin
+      # Check if project uses Yarn
+      Pod::Executable.execute_command("node", ["-e", "console.log(require.resolve('#{root}/yarn.lock'))"], true)
```

Comment:
I wonder if it wouldn't been easier to just get "__dirname" using either JavaScript, Ruby or Gradle and based on that, calculate a relative path?

E.g. `node ../../build/index.js config`